.github/workflows/ci.yml

```diff
@@ -156,18 +156,6 @@ jobs:
       - run: yarn test:conformance:generate-inputs --verify
       - run: yarn test:conformance --runner rust-
 
-  docs:
-    runs-on: ubuntu-latest
-    steps:
-      - uses: actions/checkout@v3
-      - uses: actions/setup-node@v3
-        with:
-          node-version: 16.x
-          cache: yarn
-
-      - run: corepack enable && yarn install --immutable
-      - run: yarn docs:lint
```

Comment:
do we not run prettier --check in CI anymore?

Reply:
Good catch, fixed.

In this PR I actually made prettier run on the entire repo, but forgot to add it to CI.